### DISCUSSION TITLE
Unable to DELETE the keda CRD scaledobjects.keda.sh

### DISCUSSION THREAD

**Discussion by nidiculageorge** [kedacore/keda#4897] [General]
I was trying to uninstall old keda 2.3,0 from cluster but unfortunately, I couldn't delete the following CRD

kubectl delete customresourcedefinition scaledobjects.keda.sh <UNABLE to delete>

The below ones are removed

kubectl delete customresourcedefinition clustertriggerauthentications.keda.sh
kubectl delete customresourcedefinition scaledjobs.keda.sh
kubectl delete customresourcedefinition triggerauthentications.keda.sh

Then I ran the below command which ran succsfully

helm uninstall keda -n keda 

But when I checked the crds ,I could see scaledobjects.keda.sh in terminating state,and I am unable to install any new keda version in Azure AKS ,when I run the below command

helm install keda kedacore/keda --namespace keda --version 2.10.1 --set nodeSelector."kubernetes\.io/os"=linux

I am getting the below error

**Error: INSTALLATION FAILED: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "scaledobjects.keda.sh" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "keda"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "keda"**

The above error I feel is due to the old scaledojects in the AKS cluster.

Is there a way to remove the unremoved CRD scaledobjects.keda.sh from the AKS cluster

**Comment by JorTurFer** [comment]
I think that was solved in the issue: https://github.com/kedacore/keda/issues/4898
I close this
